android/app/build.gradle

```diff
@@ -150,15 +152,22 @@ android {
             keyAlias 'androiddebugkey'
             keyPassword 'android'
         }
+        release {
+            storeFile file("$System.env.KEYSTORE_FILENAME")
+            storePassword "$System.env.KEYSTORE_PASSWORD"
+            keyAlias "$System.env.KEY_ALIAS"
+            keyPassword "$System.env.KEY_PASSWORD"
```

Comment:
I agree that this way of doing it is error prone. We should at least check whether the variables exist. Else this will fail if you do a release build on your machine outside of the CI.

Reply:
I'm not sure how we do it in Integreat, but there we provide dummy keystore which are checked into Git.